### PR TITLE
🐛  Fix select group behaviour on creatable mode

### PR DESCRIPTION
**Pull request by scopsy**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -126,6 +126,20 @@
...
+      "runtimeArgs": [
+        "run-script",
+        "start:dev"
+      ],
+      "runtimeExecutable": "npm",
+      "cwd": "${workspaceFolder}/packages/core",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "type": "pwa-node"
```
Comment (scopsy) [inline .vscode/launch.json:142]: This is not really related to this PR just a vscode configuration fix

File: apps/web/package.json [hunk 2]
```diff
@@ -89,7 +89,8 @@
     "less-loader": "4.1.0",
     "nodemon": "^2.0.15",
     "start-server-and-test": "1.11.6",
-    "webpack": "4.46.0"
+    "webpack": "4.46.0",
+    "react-error-overlay": "6.0.9"
```
Comment (scopsy) [inline apps/web/package.json:93]: this fixes the occasional error react put in console.log of `process` is not defined.

File: apps/web/src/components/templates/NotificationSettingsForm.tsx [hunk 3]
```diff
@@ -1,56 +1,38 @@
...
-        /**
-         * Quick fix to default value of notificationGroup not set when page is not refreshed
-         */
-        setTimeout(() => setValue('notificationGroup', serverGroups[0]._id), 0);
-      }
-    }
-  }, [serverGroups]);
+  >((data) => api.post(`/v1/notification-groups`, data), {
+    onSuccess: (data) => {
+      queryClient.setQueryData('notificationGroups', [...groups, data]);
```
Comment (scopsy) [inline apps/web/src/components/templates/NotificationSettingsForm.tsx:25]: Instead of refetching the entire notifications group on creation of a new one we just update the cache with the newly created item

Comment (ainouzgali) [reply apps/web/src/components/templates/NotificationSettingsForm.tsx:25]: Amazing💃🏼 Good to know

**Review by ainouzgali — APPROVED** [review]
Looks great!!
